src/Base/Unit.h

```diff
@@ -121,6 +121,7 @@ class BaseExport Unit
     static Unit ElectricalInductance;
     static Unit ElectricalConductance;
     static Unit ElectricalResistance;
+    static Unit ElectricalConductivity;
```

Comment:
Two lines above ElectricalConductance is already declared which I assume is the same thing as ElectricalConductivity